lib/spack/spack/config.py

```diff
@@ -1020,7 +1026,7 @@ def change_or_add(
 
     if found:
         update_fn(section)
-        spack.config.set(section_name, section, scope=scope)
+        set(section_name, section, scope=scope)
```

Comment:
Can we avoid, in this module, the wrapper function that use a global, and just use the global explicitly:
```python
CONFIG.set(...)
```
?